### PR TITLE
KON-224 Add Test Snippets

### PR DESCRIPTION
**Pull request by nataliapeterwas**

### CODE REVIEW

**Comment by igorwojda** [comment]
1. Rename `snippets` to `snippet`
2. Make sure that the `snippet` module is not published with the artifact

File: lib/src/snippet/kotlin/com/lemonappdev/konsist/GeneralSnippets.kt [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            }
+    }
+
+    fun `no wildcard imports allowed`() {
+        Konsist.scopeFromProject()
+            .imports()
+            .assertNot { it.isWildcard }
+    }
+
+    fun `value class has parameter named 'value'`() {
```
Comment (igorwojda) [inline lib/src/snippet/kotlin/com/lemonappdev/konsist/GeneralSnippets.kt:96]: "every value..."

File: lib/src/snippet/kotlin/com/lemonappdev/konsist/LibrarySnippets.kt [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                val type = it.name.removeSuffix("Test")
+                val sut = it
+                    .properties()
+                    .firstOrNull { property -> property.name == "sut" }
+
+                sut != null && sut.type?.name == type
+            }
+    }
+
+    fun `test classes should have all members private besides tests`() {
```
Comment (igorwojda) [inline lib/src/snippet/kotlin/com/lemonappdev/konsist/LibrarySnippets.kt:34]: Remove we have ticket for this https://lemonappdev.atlassian.net/browse/KON-243

Comment (nataliapeterwas) [reply lib/src/snippet/kotlin/com/lemonappdev/konsist/LibrarySnippets.kt:34]: This ticket is about different test - `test classes should reside in the same package as tested class`. 

I changed above test to our updated version.

**Review by igorwojda — APPROVED** [review]

**Pull request merged**